### PR TITLE
added password_generator app

### LINKED ISSUE #212: Password generator
**Project title**
Password generator

**Project Description**
Create random passwords using the password generator tool and use them during authentication

Stack:
[x] Html
[x] CSS
[x] JavaScript

### PR DESCRIPTION
**Pull request by nagasaisriya**
### Description
Password generator app
(Describe your project over here.)

### Checklist

- [x] I've been assigned an issue related to this PR.
- [x] I've used beautifiers.
- [ ] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [x] The README.md file of my projrct contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number
Fixes #212 
(Write your answer here.)

### CODE REVIEW

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Please add your project details to the index.md file.

**Comment by nagasaisriya** [comment]
@harshita2216 added the project to inex.html file, please review it and let me know if any further changes are needed:)

File: Password Generator/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <button class="btn btn-large" id="generate">
+            Generate Password
+        </button>
+
+    </div>
+    <script src="script.js"></script>
+
+</body>
+
+</html>
```
Comment (chicken-biryani) [inline Password Generator/index.html:57]: Add a new line at the eof.

File: Password Generator/script.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+function getRandomNumber() {
+    return String.fromCharCode(Math.floor(Math.random() * 10) + 48)
+}
+
+function getRandomSymbol() {
+    const symbols = '!@#$%^&*(){}[]=<>/,.'
+    return symbols[Math.floor(Math.random() * symbols.length)]
+}
```
Comment (chicken-biryani) [inline Password Generator/script.js:77]: Add a new line at the eof.

File: Password Generator/style.css [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  display: block;
+  width: 100%;
+}
+
+.setting {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 15px 0;
+}
```
Comment (chicken-biryani) [inline Password Generator/style.css:83]: Add a new line at the eof.

File: Password Generator/index.html (hunk 1 above)
Comment (nagasaisriya) [reply Password Generator/index.html:57]: added newlines to eof

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**